src/js/log_manager_helper.coffee

```diff
@@ -0,0 +1,30 @@
+_ = require "lodash"
+
+class LogManagerHelper
+  @logManagerUrl   = "//cc-log-manager.herokuapp.com/api/logs"
+  @applicationName = "HASBot-Dashoard"
+
+  constructor: (initial_data={}) ->
+    @data = _.assign initial_data,
+      application: LogManagerHelper.applicationName
+
+  log: (data) ->
+  if (typeof data == 'string')
+      data =
+        event: data
+    timestamp =
+      time: Date.now();
+    @data = _.assign(@data,data,timestamp) # update our defaults
```

Comment:
It seems we will accumulate all the parameters over time and pretty much replace default values. I'd say it can be confusing. E.g.:

1. log: {event: 'test'} => send to server: {event: 'test', application: 'HASBot-Dashoard'}
2. log: {val: '1', application: 'app'} => send to server: {val: '1', event: 'test', application: 'app'}
3. log: {event: 'test2'} => send to server {event: 'test2', val: '1', application: 'app'}
etc

What about something like:
```
finalData = _.assign({}, DEFAULT_VALUES, @basicData, data, timestamp)
```